src/api/Folder.js

```diff
@@ -237,19 +226,17 @@ class Folder extends Item {
      * @param {Function} successCallback - Function to call with results
      * @param {Function} errorCallback - Function to call with errors
      * @param {boolean|void} [forceFetch] - Bypasses the cache
-     * @param {boolean|void} [includePreview] - Optionally include preview fields
-     * @param {boolean|void} [includePreviewSidebar] - Optionally include preview sidebar fields
+     * @param {boolean|void} [includePreviewFields] - Optionally include preview fields
```

Comment:
These params are no longer present in the function signature. Also, `options` is missing.